File: CampingNeretva/UI/campingneretva_desktop/lib/models/reservation_vehicle_model.dart
import 'vehicle_model.dart';

class ReservationVehicleModel {
  final int quantity;
  final Vehicle vehicle;

  ReservationVehicleModel({required this.quantity, required this.vehicle});

  factory ReservationVehicleModel.fromJson(Map<String, dynamic> json) {
    return ReservationVehicleModel(
      quantity: json['quantity'],
      vehicle: Vehicle.fromJson(json['vehicle']),
    );
  }
}

File: CampingNeretva/CampingNeretva.Model/Models/ReservationVehicleModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CampingNeretva.Model.Models
{
    public class ReservationVehicleModel
    {
        public int VehicleId { get; set; }
        public int Quantity { get; set; }
        public VehicleModel Vehicle { get; set; }
    }
}
